stdlib/array.gr

```diff
@@ -405,6 +419,8 @@ provide let reducei = (fn, initial, array) => {
  * @param array: The array to iterate
  * @returns The new array
  *
+ * @throws InvalidArgument(String): When the combined length of the all arrays is not an integer
```

Comment:
```suggestion
 * @throws InvalidArgument(String): When the combined length of all arrays is not an integer
```

Reply:
oops, Thanks for catching these fixed them.